networking/determine-best-networking.md

```diff
@@ -1,110 +1,219 @@
+
+**What determines routability?**
+
+If you are using an overlay network for your cluster, then pod IPs are not normally routable outside of the cluster.
+
+If you aren’t using an overlay network, then whether pod IPs are routable outside of the cluster depends on what combination of CNI plugins, cloud provider integrations, or (for on-prem) BGP peering with the physical network, is being used.
+
+#### BGP
+BGP (Border Gateway Protocol) is a standards based networking protocol for sharing routes across a network.  It’s one of the fundamental building blocks of the internet, with exceptional scaling characteristics.
+
+{{site.prodname}} has built in support for BGP.  In an on-prem deployment, this allows {{site.prodname}} to peer with the physical network (typically to Top or Rack routers) to exchange routes, making a non-overlay network where pod IP addresses routable across the broader network, just like any other workload attached to the network.
+
+### About {{site.prodname}} Networking
+{{site.prodname}}’s flexible modular architecture for networking includes the following.
+
+**{{site.prodname}} CNI network plugin**
+
+The {{site.prodname}} CNI network plugin connects pods to the host network namespace’s L3 routing using a pair of virtual ethernet devices (veth pair). This L3 architecture avoids the unnecessary complexity and performance overheads of additional L2 bridges that feature in many other Kubernetes networking solutions.
+
+**{{site.prodname}} CNI IPAM plugin**
+
+The {{site.prodname}} CNI IPAM plugin allocates IP addresses for pods out of one or more configurable IP address ranges, dynamically allocating small blocks of IPs per node as required.  The result is a more efficient IP address space usage compared to many other CNI IPAM plugins, including the host local IPAM plugin which is used in many networking solutions.
+
+**Overlay network modes**
+
+{{site.prodname}} can provide both VXLAN or IP-in-IP overlay networks, including cross-subnet only modes.
+
+**Non-overlay network modes**
+
+{{site.prodname}} can provide non-overlay networks running on top of any underlying L2 network, or an L3 network that is either a public cloud network with appropriate cloud provider integration, or a BGP capable network (typically an on-prem network with standard Top-of-Rack routers).
+
+**Network policy enforcement**
+
+{{site.prodname}}’s network policy enforcement engine implements the full range of Kubernetes Network Policy features, plus the extended features of {{site.prodname}} Network Policy.  This works in conjunction with {{site.prodname}}’s built in networking modes, or any other {{site.prodname}} compatible network plugins and cloud provider integrations.
```

Comment:
range of Kubernetes network policy features, plus the extended features of {{site.prodname}} network policy. (lowercase)